appserver/admin/cli-optional/src/main/java/com/sun/enterprise/admin/cli/optional/StartDatabaseCommand.java

```diff
@@ -248,6 +250,8 @@ protected int executeCommand() throws CommandException, CommandValidationExcepti
             } else {
                 throw new CommandException(strings.get("UnableToStartDatabase", dbLog));
             }
+        } catch (CommandException ce) {
```

Comment:
This is not necessary.

Reply:
CommandException catch is required because Exception catch block throw it with new  "CommandUnSuccessful" message and validation message like invalid port or invalid db type is not available on command line

Reply:
Then IMHO you should consider throwing an Exception with a specific message containing the original CommandException.